packages/product-components/src/components/FeeRate/FeeRate.tsx

```diff
@@ -12,7 +12,8 @@ export const FeeRate = ({ feeRate, networkType }: FeeRateProps) => {
 
     return (
         <span>
-            {fee.toFixed(2)}&nbsp;{getFeeUnits(networkType)}
+            {networkType === 'bitcoin' ? fee.toFixed(2) : fee.toString()}&nbsp;
```

Comment:
I think every network/networkType should have some configuration in `networksConfig` which can also include rounding, fee units,...